Python/CMakeLists.txt

```diff
@@ -171,6 +171,14 @@ set(PRP_AVTR_HDRS
     PRP/Avatar/pySittingModifier.h
 )
 
+set(PRP_CAMERA_SRCS
```

Comment:
I'm not actually a fan of those abbreviated names, but that's probably an issue for another PR